### PR TITLE
feature/plant-model

### PR DESCRIPTION
**Pull request by brzeczkowskaw**
commented out models to be added already

### CODE REVIEW

**Review by Suegro24 — CHANGES_REQUESTED** [review]
All the data should be required, name and latinName should be unique, image could have some default value

**Review by Suegro24 — APPROVED** [review]

**Review by kami3la — APPROVED** [review]

File: server/models/example.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const mongoose = require('mongoose');
+export const mongoose = require('mongoose');
```
Comment (jedrzejratajczak) [inline server/models/example.ts:1]: imo we should export mongoose in `server.ts`

File: server/models/plant.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    toxicity: {
+        type: Toxicity,
+        required: true
+    }
+    comments: Comment*/
+})
+
+const Plant = mongoose.model('plant', PlantSchema);
+
+module.exports = Plant
```
Comment (jedrzejratajczak) [inline server/models/plant.ts]: we should use ES6 or CJS modules, not both
i would suggest using ES6

File: server/models/plant.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import {mongoose} from './example'; 
+//import {Species} from './species';
+//import {Comment} from './comment';
+//import {Toxicity} from './toxicity';
```
Comment (jedrzejratajczak) [inline server/models/plant.ts:4]: I don't think we should import comment nor toxicity
It's the same table in our erd diagram, I would include those in the same file

**Review by jedrzejratajczak — CHANGES_REQUESTED** [review]

**Review by jedrzejratajczak — APPROVED** [review]

**Pull request merged**